lib/rules/no-unknown-property.js

```diff
@@ -29,6 +29,9 @@ const DOM_ATTRIBUTE_NAMES = {
 const ATTRIBUTE_TAGS_MAP = {
   // image is required for SVG support, all other tags are HTML.
   crossOrigin: ['script', 'img', 'video', 'audio', 'link', 'image'],
+  fill: ['svg'],
```

Comment:
`fill` appears to be OK [on a number of other tags](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill), such as `path`

Reply:
This should be handled by #3385